Automattic/wp-calypso
# Domains: Transfer: Fix Privacy Protection State

(Goes together with D956-code)
This finally fixes all of the problems associated with the Privacy Protection state being broken with during transfers.

It uses a single request to handle locking, cancelling a transfer and enabling Privacy Protection altogether. This removes the visual flashing during the update caused by the race of three requests.

#### Testing
1. Ensure you're on D956-code
2. Start a transfer
3. Cancel it
4. The result should come up immediately, and if there's any error, you should see a related message to the error itself instead of a generic one.


/cc: @klimeryk @aidvu
